packages/cli/src/__tests__/commands/Generate.test.ts

```diff
@@ -37,6 +37,16 @@ describe('using cli', () => {
 
     expect(data.stdout).toContain(`I am a minimal generator`)
   }, 75_000) // timeout
+
+  it('should work with nested ', async () => {
```

Comment:
Another way to do it would be to make the test assume that it should work, and make it `it.fail` so we can just remove `fail` when it's fixed
```suggestion
  it.fail('should work with nested ', async () => {
```

Reply:
See https://jestjs.io/docs/api#testfailingname-fn-timeout